src/OSInfo.cpp

```diff
@@ -23,6 +23,20 @@
 #include "Version.h"
 
 #if defined(Q_OS_WIN)
+// regString converts a wchar_t string of size to a
+// QString. If the string contains a NUL value, that
+// NUL value will terminate the string.
+static QString regString(wchar_t *string, int size) {
+	if (size <= 0) {
```

Comment:
Oh yeah, I added the if <= 0 because of the static_cast<size_t>.
Should I drop the == 0 and make it just if (len < 0) ?